WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -189,6 +191,7 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         mSearchMenuItem = menu.findItem(R.id.menu_notifications_settings_search);
         mSearchView = (SearchView) mSearchMenuItem.getActionView();
         mSearchView.setQueryHint(getString(R.string.search_sites));
+        setSearchViewHintColor();
```

Comment:
I noticed that you changed this from your first implementation in 1989207a44462b8d0ec6342793cd560bad725964. I think we should do it that way and make it a `static` too. Possibly shallow reasons:

1. A `private void` can indicate that this method can and should be used multiple times but it doesn't really need to be. An instance method also adds a bit of cognitive overhead to analyze how the _state_ changes in this class.  
2. A `static void` would imply that it's a utility method, which this one is
3. It's arguably better for readability when you're scanning the contents of `onCreateOptionsMenu`

Sample untested code: 

```java
    private static void setSearchViewHintColor(@NonNull SearchView searchView, @ColorRes int colorResId) {
        final Context context = searchView.getContext();
        if (context != null) {
            ((EditText) searchView.findViewById(androidx.appcompat.R.id.search_src_text))
                    .setHintTextColor(ContextCompat.getColor(context, colorResId));
        }
    }
```

What do you think? 🙂

Reply:
Thanks. Yes, your reasoning is sound and I was thinking of making it a utility method at first hence the first implementation. Since its a utility method wouldn't it make sense to put it somewhere else so if another person wants to utilize it, it's accessible? Let me know what you think about that.

Reply:
>  Since its a utility method wouldn't it make sense to put it somewhere else so if another person wants to utilize it, it's accessible?

I'm okay with keeping it private here and make it public later when we violate the [Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)). But up to you!

Reply:
Okay, thanks for citing that rule! Yes, it's best to make it public at that time 👍

Reply:
Resolved here fe68ca6